odkx-src/spelling_wordlist.txt

```diff
@@ -91,6 +91,7 @@ esthetics
 ethiopian
 extensibility
 fallback
+flavor
```

Comment:
flavor should not need to be added to the list. It was a style check problem disliking the usage of "flavour" in favor of "flavor".